src/ros_commo.py

```diff
@@ -444,10 +445,8 @@ def __init__(self):
 			self.language_perceived_text_cb)
 
 		# Chat infrastructure text.
-		# from chatbot.msg import ChatMessage
-		#rospy.Subscriber("/robot/speech", chatbot/ChatMessage,
-		#rospy.Subscriber("chatbot_speech", chatbot/ChatMessage,
-		#	self.chat_perceived_text_cb)
+		rospy.Subscriber("chatbot_speech", ChatMessage,
```

Comment:
It is good practice not to break other users by checking in code that adds a brand new dependency on something that the user does not have, and is also (very) difficult to install.  The first time I tried, it took me a month to install these dependencies.  To ask complete newbies to do this means that it is defacto impossible.